File: Code/App/Hospital/Ward/ViewModels/WardPatientDeclarationViewModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using BusinessLogic.Models;

namespace Ward.ViewModels
{
    public class WardPatientDeclarationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public WardPatientCurrentDieseaseViewModel CurrentDiesease { get; set; }
    }

}

File: Code/App/v2/Ward/Scripts/App/Models/WardPatientDeclarationViewModel.js
function WardPatientDeclarationViewModel(data) {
    var self = this;

    this.PatientInfo = new PatientsModel(data.PatientInfo);
    this.CurrentDiesease = new
        WardPatientCurrentDieseaseViewModel(data.CurrentDiesease);
    this.examinationDescriptionBlood = ko.observable();
    this.examinationDescriptionUSG = ko.observable();
    this.examinationDescriptionRTG = ko.observable();

};